ricecooker/chefs.py

```diff
@@ -290,16 +293,15 @@ def run(self, args, options):
 
     def main(self):
         args, options = self.parse_args_and_options()
+        if args['stage_deprecated']:
+            config.LOGGER.warning('DEPRECATION WARNING: --stage is now default, so the --stage flag has been deprecated and will be removed in ricecooker 1.0.')
```

Comment:
@kollivier Could you move this warning to here:
https://github.com/learningequality/ricecooker/blob/master/ricecooker/chefs.py#L94

this way all the subclasses (SushiChef, JsonTreeChef, etc) will behave the same way.